src/lib.rs

```diff
@@ -1207,6 +1221,11 @@ impl Settings {
         self.idle_timeout_ms = value;
         self
     }
+    pub fn set_datagram_receive_enabled(&mut self, value: bool) -> &mut Settings {
+        self.is_set_flags |= (value as u64) << 27;
```

Comment:
This we really be using `value` here? You are setting the receive enabled flag (to true or false) so I think this should always be set.

Reply:
For instance, look at the functions above, such as `set_idle_timeout_ms`. The `is_set_flags` is always set because you are modifying the actual setting value (independent of it being true or false). So you should always be setting the appropriate bit in `is_set_flags` here. For instance:
```suggestion
        self.is_set_flags |= 1 << 27;
```

Reply:
Aah, finally got it, sure you are right, will change.